src/WebJobs.Extensions.DurableTask/ITokenSource.cs

```diff
@@ -6,9 +6,7 @@
 namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
 {
     /// <summary>
-    /// Provides functionality available to sources of tokens.
-    /// Custom implementation of this interface must include the use
```

Comment:
Did you remove the documentation about `[DataContract]`/`[DataMember]`  because it was too much of an implementation detail, or does the use of TokenSourceConverter remove that requirement?

If the latter, we should remove `[DataContract]` and `[DataMember]` from `MockTokenSource` in DurableHttpTests.cs

Reply:
The `TokenSourceConverter` removes this requirement. However, I suspect I'll still need `[DataContract]` and `[DataMember]` on the `MockTokenSource` since it uses standard serialization. It's not a pattern we want others to use though, so I don't want to document it.

Reply:
Do we not want customers to be able to provide their own customer `ITokenSource` implementations? Or just not ones with state?

Reply:
I guess I hadn't really thought about that. My thinking was that we'd want to come up with some mechanism that works for out-of-proc languages too, but I guess the current model works well-enough for .NET. I would still rather we not include the custom implementation comments here. I think it would be better to put that information in official docs.